client/app/widgets/Comments/index.jsx

```diff
@@ -104,6 +104,7 @@ export const Comments = ({ comments, formProps }: Props): Node => {
       deleteAction,
     } = myComment;
     const author = <a href={`/profile?uid=${commentByUid}`}>{commentByName}</a>;
+    const commentStoryName = `from comment from ${commentByName}`;
```

Comment:
We use i18n for text in our app. You'll want to add a key in [this part of en.yml](https://github.com/ifmeorg/ifme/blob/main/config/locales/en.yml#L113) and add it to the other translation files. You can use Google translate for this, and we'll verify the translations later.

I should create two keys like the following:

```
report_comment_by: 'Report comment by %{name}`
delete_comment_by: 'Delete comment by %{name}`
```